code/go/0chain.net/chaincore/chain/protocol_view_change_integration_tests.go

```diff
@@ -46,11 +54,16 @@ func (c *Chain) SetupSC(ctx context.Context) {
 				case reg := <-isRegisteredC:
 					if reg {
 						logging.Logger.Debug("SetupSC - node is already registered")
+						func() {
```

Comment:
you don't need wrapping function here, also if you recover you should do something with error you catch